docs/codeql/codeql-cli/analyzing-databases-with-the-codeql-cli.rst

```diff
@@ -239,18 +240,23 @@ recursively, so any queries contained in subfolders will also be executed.
 
    Important
 
-   You shouldn't specify the root of a :doc:`QL pack
-   <about-ql-packs>` when executing ``database analyze``
+   You shouldn't specify the root of a :doc:`CodeQL pack
```

Comment:
Yes. I'll change the order. 

Custom packs might, or might not contain custom special queries. If you run the pack, it's generally safer since you don't know unless you inspect the pack. It's always safer to run the pack instead of the directory since there's no danger of running special queries.

I'll change it to say something like that.

Reply:
Actually, I take that back, this paragraph really is focused on core packs.